### DISCUSSION TITLE
Need to install Xiaomi Cloud Map Extractor when using Dream-vacuum

### DISCUSSION THREAD

**Discussion by z1022** [Tasshack/dreame-vacuum#176] [Q&A] [answered]
use dreame-vacuum need to install Xiaomi Cloud Map Extractor ?
It is because I cannot show the pic in xiaomi-vacuum-map-card.

Thanks

**Comment by Tasshack** [comment]
You don't need to install the map extractor, integration should render the map but if you can't see the camera entity there may be a problem with your setup.

**Reply by z1022** [reply]
see  camera.xiaomi_robot_vacuum_s10_map entity in developer tools->state

procedure

1. Install Dreame Vacuum
2. confguration use With map
3. install Xiaomi Vacuum Map Card

still cannot show map

did not install Xiaomi MIoT

Need to inpout Token ?
Any setup need to notice

**Reply by Tasshack** [reply, marked as answer]
You dont need to install anything other than [xiaomi-vacuum-map-card](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card).
Please check this issue https://github.com/Tasshack/dreame-vacuum/issues/166
